packages/blade/src/components/Icons/_Svg/Rect/Rect.native.tsx

```diff
@@ -1,6 +1,6 @@
 import type { ReactElement } from 'react';
 import { Rect as RectNative } from 'react-native-svg';
-import type { RectProps } from './Rect';
+import type { RectProps } from './Rect.d';
```

Comment:
At some places you're using `.d`. Are they meant to be moved to `types` too?

Reply:
I guess we can do it for the consistency sake. But they didn't really cause any issues so didn't bothered migrating them.  
I only faced issues in few of the components like StyledButton while I was migrating (tho that time didn't figure out the module resolution)

Reply:
I see, maybe we can have a followup issue to keep track of this for consistency